packages/eos-components/src/components/PlayButton.vue

```diff
@@ -76,6 +77,14 @@ export default {
           return 'VideoIcon';
       }
     },
+    iconSize() {
+      switch (this.size) {
+        case 'xs':
+          return 20;
+        default:
+          return 24;
```

Comment:
Is this the first time that these constants are being used? It would be better to move them to `constants.js`.